CONTRIBUTING.md

```diff
@@ -145,14 +145,17 @@ browse the [`docs/ directory`](https://github.com/Leaflet/Leaflet/tree/main/docs
 choose a certain file for editing, click the Edit button, make changes and follow instructions from there.
 Once it gets merged, the changes will immediately appear on the website.
 
+To work on the documentation locally ensure you have Ruby installed. You can download it from the [Ruby website](https://www.ruby-lang.org/en/downloads/) or use [`rbenv`](https://github.com/rbenv/rbenv) (recommended).
+
+You'll need to install the same Ruby version as specified in [`.ruby-version`](./docs/.ruby-version). If you are using `rbenv` you can install this by running `rbenv install` from the `docs/` directory.
+
```

Comment:
Ruby 2 is only necessary for `github-pages` but not for running `jekyll`. I know the user should use the same as Github but I think the most users will install the newest version or already have version 3 because Ruby 2 is not maintained anymore. Can we support Ruby 2 and 3 on local for `jekyll`?